e2e/test/scenarios/dashboard-filters/reproductions/16663-filters-can-stay-in-url.cy.spec.js

```diff
@@ -0,0 +1,64 @@
+import {
+  restore,
+  popover,
+  editDashboard,
+  saveDashboard,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+
+const { ORDERS_ID } = SAMPLE_DATABASE;
+
+const questionDetails = {
+  query: {
+    "source-table": ORDERS_ID,
+  },
+};
+
+const parameters = [
+  {
+    name: "Quarter and Year",
+    slug: "quarter_and_year",
+    id: "f8ae0c97",
+    type: "date/quarter-year",
+    sectionId: "date",
+  },
+];
+
+const dashboardDetails = { parameters };
+
+describe("issue 16663", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should remove filter value from url after going to another dashboard (metabase#16663)", () => {
+    cy.createQuestionAndDashboard({ questionDetails, dashboardDetails }).then(
+      ({ body: dashboardCard }) => {
+        const { dashboard_id } = dashboardCard;
+
+        visitDashboard(dashboard_id);
+      },
+    );
+
+    editDashboard();
+
+    cy.get("main header").find(".Icon-gear").click();
+
+    cy.findByLabelText("subscriptions sidebar")
+      .findByText("No default")
+      .click();
+
+    popover().contains("Q1").click();
+
+    saveDashboard();
```

Comment:
All these actions to set the default parameter value could be avoid by setting the default value in dashboard parameter when creating the dashboard directly.

```diff
const parameters = [
  {
    name: "Quarter and Year",
    slug: "quarter_and_year",
    id: "f8ae0c97",
    type: "date/quarter-year",
    sectionId: "date",
+   default: "Q1-2023",
  },
];
```

This should speed up the test. From my machine, it reduces the time from 8-9s to 6s. That's not huge by any mean, but we should avoid using the UI to prepare the data when possible.

Reply:
changed, thanks!